src/lib/subgraph.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+import yargs from './getClargs';
+import { graphEndpoints } from './constants'
+
+
+const sortByTime = (a: any,b: any): number => {
+    if(a.blockNumber === b.blockNumber) {
+        return a.logIndex - b.logIndex
+    }
+    return a.blockNumber - b.blockNumber
+}
```

Comment:
what type is a and b? If those are BigNumber or String the sorting wouldn't work properly

Reply:
easy way is just cast them to bignumber anyway

Reply:
Add a new interface for this

Reply:
are you sure they are number? just asking I do not know the answer

Reply:
Yes, checked the return, they are uint directly, and their number should always less than int32.

Reply:
But to avoid the future changes, just add [number cast](https://github.com/OffchainLabs/arb-token-lists/pull/45/commits/ce0b130aa64391e23f2fa67c5e7dce9f4428f9c5) to it. (This value should always be able to cast to number)